server/cluster/cluster.go

```diff
@@ -1121,7 +1122,7 @@ func (c *RaftCluster) processRegionHeartbeat(ctx *core.MetaProcessContext, regio
 	tracer.OnCollectRegionStatsFinished()
 	if c.storage != nil {
 		if saveKV {
-			ctx.TaskRunner.RunTask(
+			ctx.StatisticsRunner.RunTask(
```

Comment:
`SaveRegionToKV` is put into the statistics runner, the runner name might be strange. But I cannot come up with a better name.

Reply:
How about just calling it `MiscRunner`?